integration-tests/ci-visibility/jestEnvironmentBadInit.js

```diff
@@ -3,4 +3,5 @@ require('dd-trace').init({
   service: 'dd-trace-bad-init'
 })
 
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Similarly, if this is allowed in the project, why not add it as a global rule?

Reply:
It's some junk for a test that we otherwise wouldn't want i real tracer code

Reply:
Could always add a .eslintrc override at the root folder of the tests.